app/views/content/is-teaching-right-for-me/teacher-pay-and-benefits.md

```diff
@@ -148,7 +148,7 @@ Or find out how you can earn an unqualified teacher salary while training to get
 
```

Comment:
Am pretty sure we said we'd do unqualified separately, but noticed they're in the list of variables below so flagging just in case!

Reply:
Quite right - I've ticketed all of these up in a separate unqualified salary ticket - I'll create a PR for this on Monday as soon as this main one goes out!